docs/src/architecture/transactions.md

```diff
@@ -1,61 +1,60 @@
 
-It looks like 
-
-<p align="center">
-    <img src="../diagrams/architecture/transaction/Transaction.png">
-</p>
+A transaction must include an executable program in addition to inputs and outputs. The transaction program has a well-defined structure and must perform the following functions:
 
-In addition to specifying inputs and outputs, a transaction must also include an executable program. The transaction program has a well-defined structure which must do the following:
+1. Create a uniform vault for all inputs.
+2. Run scripts for all input notes (scripts are run one after the other).
+3. Run a user-defined script if desired.
+4. Create a uniform vault for all outputs.
+5. Ensure that the assets in the unified input and output vaults are the same.
 
-1. Build a single unified vault for all inputs.
-2. Execute scripts of all input notes (the scripts are executed one after another).
-3. Execute an optional user-defined script.
-4. Build a single unified vault for all outputs.
-5. Make sure the unified input and output vaults contain the same set of assets.
+The final point verifies that no assets are created or destroyed as a result of the transaction.
 
-The last point ensures that a transaction does not create or destroy any assets.
+A user-defined transaction script can also be included in a transaction.
 
-A transaction can also include a user-defined transaction script. <Some more info here is missing>
+## Execution Steps 
+There are various steps involved in transaction execution. These stages are detailed here (in a somewhat different order):
 
-## Execution steps of a transaction
-Transaction execution consists of several steps. These steps (grouped slightly differently) are described below:
+1. **Prologue**: In this stage, we create a unified vault for all transaction inputs (accounts and notes).
+2. **Execution**: In this stage, we first execute scripts for all input notes (one by one), and then an optional user-defined script (called tx script) is executed.
+3. **Epilogue**: During this stage, we create a unified vault of all transaction outputs (accounts + notes) and ensure that it contains the same assets as the input vault.
 
-1. **Prologue**: during this step we build a single unified vault all all transaction inputs (account + notes).
-2. **Execution**: during this step we first execute scripts of all input notes (one after another), and then execute an optional user-defined script (called tx script).
-3. **Epilogue**: during this step we build a single unified vault of all transaction outputs (account + notes), and make sure it contains the same assets as the input vault.
+### Example: Transaction that sends assets out of a wallet (creating a note):
 
-### Example 1: transaction that sends assets out of a wallet (creating a note):
+This transaction does not consume any notes but has transaction script which calls the send function. Send function then creates an output note (not pictured).
```

Comment:
Nit: I think here is an article missing `This transaction does not consume any notes but has **a** transaction script which calls the send function.`